lib/constants/constants.go

```diff
@@ -230,6 +230,12 @@ const (
 	// OneshotService is a service that executes one time
 	OneshotService = "oneshot"
 
+	// SimpleService is a simple service that is recommended for long running processes
```

Comment:
I think a link to the systemd docs as a reference to service types would be useful.
Also, I think these could be more appropriate in [lib/system/service](https://github.com/gravitational/gravity/tree/bernard/master/upgrade-agent/lib/system/service) but it's debatable.